presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,164 @@
+
+## Local File Connector Changes
+
+* Change `timestamp` column type to `TIMESTAMP WITH TIME ZONE`. ({issue}`4752`)
+
+## MySQL Connector Changes
+
+* Improve performance of aggregation queries by computing aggregations within MySQL database.
+  Currently, the following aggregate functions are eligible for pushdown: `count`,  `min`, `max`,
+  `sum` and `avg`. ({issue}`4138`)
+
+## Oracle Connector Changes
+
+* Add `oracle.connection-pool.inactive-timeout` config property to specify how long
+  pooled connection can be inactive before it is closed. ({issue}`4779`)
+* Add support for database globalization. ({issue}`4775`)
+* Resilience to momentary JDBC connection authentication issues with Oracle. ({issue}`4947`)
+* Add possibility to force mapping of certain types to ``varchar``. This can be enabled
+  by setting ``jdbc-types-mapped-to-varchar`` to comma-separated list of type names. ({issue}`4955`)
+* Properly pushdown predicates for queries where number of conjuncts exceeded 1000. Previously query
+  would fail in such case. ({issue}`4918`)
+
+## Phoenix Connector Changes
+
+* Fix overwriting of former value when insert is applied without specifying that column. ({issue}`4670`)
+
+## Pinot Connector Changes
+
+* Add support for `REAL` and `INTEGER` types. ({issue}`4725`)
+* Add support for functions in pass-through queries. ({issue}`4801`)
+* Enforce a limit on the number of rows fetched from Pinot. This can be configured via the `pinot.max-rows-per-split-for-segment-queries`
+  config property. ({issue}`4723`)
+* Fix incorrect results for `count(*)` queries. ({issue}`4802`)
+* Fix incorrect results for queries involving {func}`avg` over columns of type `long`, `int`, or `float`. ({issue}`4802`)
+* Fix incorrect results when columns in pass-through query do not match selected columns. ({issue}`4802`)
+
+## Prometheus Connector Changes
+
+* Change the type of the `timestamp` column to `TIMESTAMP(3) WITH TIME ZONE` type. ({issue}`4799`)
+
+## PostgreSQL Connector Changes
+
+* Improve performance of aggregation queries with predicates (the `WHERE` clause) by computing both filtering and
+  aggregations within PostgreSQL database where possible. ({issue}`4111`)
+* Fix handling of PostgreSQL arrays when `unsupported-type-handling` is set to `CONVERT_TO_VARCHAR`. ({issue}`4981`)
+
+## Raptor Connector Changes
+
+* Change timestamp internal storage to match new Presto semantics. Tables containing timestamp data written by a client not running in UTC
+  will see timestamps adjusted to UTC. ({issue}`4799`)
+* Remove the `storage.shard-day-boundary-time-zone` configuration property, which was used to work around legacy timestamp semantics in
+  Presto. ({issue}`4799`)
+
+## Redis Connector Changes
+
+* Preserve time zone when parsing `TIMESTAMP WITH TIME ZONE` values. ({issue}`4799`)
+
+## SPI Changes
+
+* The `TIMESTAMP` type is encoded as a number of fractional seconds from `1970-01-01 00:00:00`, where every day is defined to contain
```

Comment:
is "fractional seconds" a valid term?

Reply:
Yes (vs "whole seconds"). What feels weird about it?